### PR TITLE
f2 1.7.2 (new formula)

### PR DESCRIPTION
**Pull request by simono**
A command-line batch renaming tool.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/f2.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  homepage "https://github.com/ayoisaiah/f2"
+  url "https://github.com/ayoisaiah/f2/archive/v1.7.2.tar.gz"
+  sha256 "5c44d9d7bbd428e7c0d1a37118fbfec47b38dbb708501a2b3adc4f4da3e049ef"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOBIN"] = bin
+    system "go", "install", "./..."
```
Comment (SMillerDev) [inline Formula/f2.rb:12]: I think this would be better
```suggestion
    system "go", "build", *std_go_args
```

File: Formula/f2.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+class F2 < Formula
+  desc "Command-line batch renaming tool"
+  homepage "https://github.com/ayoisaiah/f2"
+  url "https://github.com/ayoisaiah/f2/archive/v1.7.2.tar.gz"
+  sha256 "5c44d9d7bbd428e7c0d1a37118fbfec47b38dbb708501a2b3adc4f4da3e049ef"
+  license "MIT"
+
```
Comment (chenrui333) [inline Formula/f2.rb:7]:
```suggestion
  license "MIT"
  head "https://github.com/ayoisaiah/f2.git", branch: "master"

```

Comment (simono) [reply Formula/f2.rb]: Added. Thanks @chenrui333

File: Formula/f2.rb (hunk 1 above)
Comment (simono) [reply Formula/f2.rb]: Thanks @SMillerDev 
I added this and got an error:
```shell
==> go build                                                                                                                                                                                                                                  
Last 15 lines from /Users/simon/Library/Logs/Homebrew/f2/01.go:                                                                                                                                                                               
2021-08-25 09:42:50 +0200                                                                                                                                                                                                                     
                                                                                                                       
go                                                                                                                     
build                                                                                                                  
-trimpath                                                                                                              
-o=/usr/local/Cellar/f2/1.7.2/bin/f2                                                                                   
                                                           
no Go files in /private/tmp/f2-20210825-78983-bl7l9r/f2-1.7.2 
```

With `system "go", "build", *std_go_args, "./cmd..."` it works. Taken from here: https://github.com/ayoisaiah/f2/blob/master/Makefile#L10

**Review by chenrui333 — APPROVED** [review]
lgtm

**Pull request closed without merging**